resctrl/utils.go

```diff
@@ -114,60 +117,79 @@ func prepareMonitoringGroup(containerName string, getContainerPids func() ([]str
 		return "", fmt.Errorf("couldn't obtain %q container pids: there is no pids in cgroup", containerName)
 	}
 
+	if !inHostNamespace {
+		processPath = "/rootfs/proc"
+	}
+	var processThreads []string
+	for _, pid := range pids {
+		processThreads, err = getAllProcessThreads(filepath.Join(processPath, strconv.Itoa(pid), processTask))
+		if err != nil {
+			return "", err
+		}
+	}
+
```

Comment:
Can you explain what are the benefits of this approach? As far as I understand (and I might be wrong) you extracted obtaining task IDs to separate loop, but the result will be the same.

Reply:
Now threads are extracted earlier and passed as param to the `findGroup` func, which previously accepted pids resulting in the incomplete list